File: code/c/Code_2_02_2.c
#include <stdio.h>

int main() {
	int a, b;
	scanf("%d%d", &a, &b); // 輸入 a 及 b
	printf("%d\n", a & b); // 輸出 a AND b的值
	printf("%d\n", a | b); // 輸出 a OR b的值
	printf("%d\n", a ^ b); // 輸出 a XOR b的值
	return 0;
}

File: code/java/Code_2_02_2.java
import java.util.*;

class Code_2_02_2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt(); // 輸入 a
		int b = sc.nextInt(); // 輸入 b
		System.out.println(a & b); // 輸出 a AND b的值
		System.out.println(a | b); // 輸出 a OR b的值
		System.out.println(a ^ b); // 輸出 a XOR b的值
	}
};
